build/source/engine/systemSolv.f90

```diff
@@ -2708,18 +2308,15 @@ real(dp),intent(out)           :: grad(:)       ! gradient of the function vecto
      do iState=1,nState
       do jState=max(1,iState-ku),min(nState,iState+kl)
        aJac_test(kl + ku + 1 + jState - iState, iState) = aJac(jState,iState)
-       if(iState<6 .or. jState<6) write(*,'(2(i4,1x),e11.5)') jState,iState,aJac(jState,iState)
       end do
      end do
      print*, '** test banded analytical Jacobian:'
      write(*,'(a4,1x,100(i11,1x))') 'xCol', (iLayer, iLayer=iJac1,iJac2)
-     do iLayer=kl+1,nBands; write(*,'(i4,1x,100(e11.5,1x))') iLayer, aJac_test(iLayer,iJac1:iJac2); end do
-     !pause
+     do iLayer=kl+1,nBands; write(*,'(i4,1x,100(e17.10,1x))') iLayer, aJac_test(iLayer,iJac1:iJac2); end do
```

Comment:
Follow SUMMA code conventions for multiple statements on a single line (i.e. don't unless error handling or case statements).